static/scripts/draw.ts

```diff
@@ -1,4 +1,7 @@
-(function draw() {
+//@ts-nocheck
+//@todo make it ts compatible
```

Comment:
I agree that it is out of scope to refactor this file.

Reply:
would be better if u could update this one, I'm not fully understanding the flow of `draw.js` @pavlovcik

Reply:
It's ancient code I wrote so it might be okay to just not worry about it. Its purely cosmetic and has no utility from the business logic perspective.